### ISSUE TITLE
Add Ability to open local files

### ISSUE THREAD

**Issue by jakegrabowski** [DannyBen/snapcrawl#45]
If is not much work it would be nice to have possibility opening files directy from hdd.

Current behavior
```
docker run --rm -it --volume "$PWD:/app" dannyben/snapcrawl file:///Users/jake/Dropbox/Docker/snapcrawl/index.html
 INFO : processing http://file:///Users/jake/Dropbox/Docker/snapcrawl/index.html, depth: 0
ERROR : http error on http://file:///Users/jake/Dropbox/Docker/snapcrawl/index.html - SocketError: Failed to open TCP connection to file:80 (getaddrinfo: Name or service not known)
```

**Comment by DannyBen** [comment]
Yes, perhaps. I might take a look at implementing it.

An alternative would be to run a simple lightweight static web server. 
Since I see you are using docker already, its just a matter of running another container.

One such server is [httpme](https://github.com/dannyben/httpme) which can also be executed via docker:

```
$ docker run --rm -it -p 3000:3000 -v $PWD:/docroot dannyben/httpme
```

And then you snapcrawl on <http://localhost:3000>

**Comment by jakegrabowski** [comment]
Thanks for posting example solution for docker. Please note that users without docker still are unable to crawl local files.

**Comment by jakegrabowski** [comment]
Yeah I know. It's not priority for me either. Just took a time to make note about this use case because you took time to publish your work.
